### DISCUSSION TITLE
Tag render number incorrect

### DISCUSSION THREAD

**Discussion by teddythinh** [squidfunk/mkdocs-material#6209] [Q&A] [answered]
I found out that when I make a tag for a version (e.g.: 5.10), it doesn't render the number 10, it renders the number 1.
<img width="70" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/85171754/490e82ae-3bb1-42f7-a6a9-16c9555f3a56">
![image](https://github.com/squidfunk/mkdocs-material/assets/85171754/ef5d2e86-6a56-425e-95b9-5ce65cd80828)
I've checked with other numbers like 20, 300, 4000. I don't know if it's a bug or a feature 😸

**Comment by iBug** [comment, marked as answer]
YAML interprets that as numbers, so 4.10 == 4.1 makes total sense. Add quotes around it to force interpretation as string.

**Reply by teddythinh** [reply]
Thank you for the fast reply, it works now.
